src/relayedTransactionV2Builder.spec.ts

```diff
@@ -0,0 +1,41 @@
+import {loadTestWallets, TestWallet} from "./testutils";
+import {RelayedTransactionV2Builder} from "./relayedTransactionV2Builder";
+import {Address} from "./address";
+import {TransactionPayload} from "./transactionPayload";
+import {assert} from "chai";
+
+describe("test relayed v2 transaction", function () {
+    let alice: TestWallet, bob: TestWallet;
+
+    before(async function () {
+        ({alice, bob} = await loadTestWallets());
+    });
+
+    it("should compute relayed v2 tx", async function () {
+        this.timeout(20000);
+
+        let networkConfig = {
+            MinGasLimit: 50_000,
+            GasPerDataByte: 1_500,
+            GasPriceModifier: 0.01,
+            ChainID: "T"
+        };
+
+        const builder = new RelayedTransactionV2Builder();
+        const relayedTxV2 = await builder.buildRelayedTransactionV2(
+            bob.signer,
+            Address.fromBech32("erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u"),
+            15,
+            new TransactionPayload("getContractConfig"),
+            60_000_000,
+            "T",
+            networkConfig);
+        await alice.signer.sign(relayedTxV2);
+
+        assert.equal(
+            relayedTxV2.getData().toString(),
```

Comment:
```suggestion
        assert.equal(relayedTxV2.getData().toString(),
```

Reply:
it is easier to read if there are more lines